.azure/templates/now-deployment.yml

```diff
@@ -12,13 +12,16 @@ parameters:
 - name: 'nowProjectId'
   type: string 
   default: false
+- name: 'prod'
+  type: string
+  default: ""
 
 steps:
   - task: DownloadBuildArtifacts@0
     inputs:
       artifactName: '${{ parameters.artifactName }}'
 
-  - script: NOW_ORG_ID=$org NOW_PROJECT_ID=$projectId npx now --token=$token
+  - script: NOW_ORG_ID=$org NOW_PROJECT_ID=$projectId npx now --token=$token ${prod:+--prod}
```

Comment:
So does `--token` take two different tokens?

Reply:
No, this uses [Shell Parameter Expansion](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html) to apply the `--prod` flag for `next` deployments.